contents/docs/experiments/exposures.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: Exposures
+sidebar: Docs
+showTitle: true
+---
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+import { ProductScreenshot } from 'components/ProductScreenshot'
+
+Exposures are the foundation of experiment analysis in PostHog. A user must be **exposed** to your experiment before they can be included in any metric calculations. Understanding how exposures work is crucial for running successful experiments.
+
+## What is an exposure?
+
+An exposure occurs when a user encounters the part of your product where the experiment is running. This is the moment they become a participant in your experiment and start contributing to your metrics.
+
+**For most users**: If you're using PostHog feature flags with our SDKs, exposures are tracked automatically. When you call `getFeatureFlag()`, PostHog sends a `$feature_flag_called` event with all the necessary properties. You don't need to do anything extra.
```

Comment:
I think there are some other calls that also trigger `$feature_flag_called` depending on the SDK. I recall seeing some other ones for React.

Not sure if we want to include those here or link to a full list somewhere, but feels important